docs/faqs.md

```diff
@@ -0,0 +1,153 @@
+# Frequently asked questions
+
+## Is *Jeff* meant to be an intermediate format that quantum software packages manipulate?
+
+No. *Jeff* is designed purely to be an exchange format. It is expected that software
+frameworks map **Jeff** input to their own intermediate representations for internal manipulation.
+
+## Why design a non-human readable format?
+
+The main goal is to design an *efficient* exchange format that best represents the information
+needed for quantum optimizations by compilers, and the most efficient approach is a binary,
+non-human readable format.
+
+[Protocol buffers](https://en.wikipedia.org/wiki/Protocol_Buffers) (and derivatives such as Cap'n Proto) are a binary
+data format initially developed by Google that has been used for similar problems across the
+classical domain, and was chosen for *Jeff*.
+
+Human-readable formats have been quite common in the quantum space thus far, but in addition we see
+two major reason why they might make less sense going forward:
+
+- We are only just beginning to scale quantum software to FTQC regimes, so have not experienced the
+  performance issues associated with (re-)parsing human readable formats, compared to formats
+  designed for machine-readability.
+
+- the tooling around IR and exchange formats has not been mature enough, and the best way to
+  debug/inspect has often been to simply open and read the file. Improving tooling for debugging
+  and visualization should help rectify this.
```

Comment:
Is the idea here that a debugging tool might also parse Jeff, and display some structured program representation?  Would that require either (a)  tools to be implemented by the individual compilers / providers, or (b) tools that allow visualization and/or debugging at a universal but more abstract level?